main.go

```diff
@@ -22,11 +19,45 @@ import (
 
 // Toggl service constants
 const (
-	TogglAPI       = "https://api.track.toggl.com/api/v8"
+	TogglAPI       = "https://api.track.toggl.com/api/v9"
 	ReportsAPI     = "https://api.track.toggl.com/reports/api/v2"
 	DefaultAppName = "go-toggl"
+
+	workspaceURLPattern = "/workspaces/%d"
```

Comment:
Maybe push this up and call it `WorkspaceURLPattern` for consistency, unless it needs to be separate.

Reply:
Oh, wait, never mind. I totally forgot that go exports symbols based on case. 🤦‍♂️